ciri/api/http.c

```diff
@@ -346,14 +346,16 @@ static inline retcode_t process_get_trytes_request(iota_api_http_t *const http,
     goto done;
   }
 
-  // TODO Deserialize request
+  if ((ret = http->serializer.vtable.get_trytes_deserialize_request(&http->serializer, payload, req)) != RC_OK) {
+    goto done;
+  }
 
   if ((ret = iota_api_get_trytes(http->api, tangle, req, res, &error)) != RC_OK) {
     error_serialize_response(http, &error, out);
```

Comment:
it's not related to this PR, but need to get the error code from `error_serialize_response`.

Reply:
It would override the error from the API which IMO is more important